BearlyHumans/squirrelville
# Humans animation

This Pull request adds the following 
* Different animations for the various human movements 


Steps to test: 
* Be in Lachie Scene
* Test walking - look if human is correctly walking

https://user-images.githubusercontent.com/20939519/133872488-0109708f-5a07-4f76-b05f-d836077d292c.mp4


* Test Running - walk into human range and see if human runs after player

https://user-images.githubusercontent.com/20939519/133872493-a725aad5-e864-4e1a-86ba-077348f74856.mp4

* Test Stun -  Get caught and see if human stomps

https://user-images.githubusercontent.com/20939519/133872498-8b5d4288-5c28-4b96-b1e8-18ce580bdc78.mp4


* Test Pick up - Have food when caught and see if human picks up each piece

https://user-images.githubusercontent.com/20939519/133872509-124ee196-9ade-4980-9662-21698450c3ab.mp4

* Test Eating - Get caught with food and see if human eats it


https://user-images.githubusercontent.com/20939519/133872515-25326981-d321-4aab-8f08-fcfb5b3dba15.mp4


* Test Drop -  Get caught with food and see if human walks to bin to drop it off

https://user-images.githubusercontent.com/20939519/133872518-0b7377a6-87e8-4d4a-9b86-d993d13e5a0c.mp4

Trello card:
https://trello.com/c/NGDFv7UB
